charts/newrelic-k8s-metrics-adapter/ci/test-values.yaml

```diff
@@ -7,6 +7,7 @@ verboseLog: false
 config:
   accountID: 111
   region: EU
+  nrdbClientTimeoutSeconds: 60
```

Comment:
any reason we don't use our default value for testing?

Reply:
no, let me update it to 30